doc/sources/algorithms.rst

```diff
@@ -362,6 +364,174 @@ Nearest Neighbors
+
+       - ``algorithm`` != `'brute'`
+       - ``weights`` = `'callable'`
+       - ``metric`` not in [`'euclidean'`, `'manhattan'`, `'minkowski'`, `'chebyshev'`, `'cosine'`]
+       - ``predict_proba`` method not supported
+     - Only dense data is supported
+   * - `LogisticRegression`
+     - All parameters are supported except:
+
+       - ``solver`` != `'newton-cg'`
+       - ``class_weight`` != `None`
+       - ``sample_weight`` != `None`
+       - ``penalty`` != `'l2'`
+     - Only dense data is supported
+
+Regression
+**********
+
+.. list-table::
+   :widths: 10 30 20
+   :header-rows: 1
+   :align: left
+
+   * - Algorithm
+     - Parameters & Methods
+     - Data formats
+   * - `RandomForestRegressor`
+     - All parameters are supported except:
+
+       - ``warm_start`` = `True`
+       - ``cpp_alpha`` != `0`
+       - ``criterion`` != `'mse'`
+       - ``oob_score`` = `True`
+       - ``sample_weight`` != `None`
+     - Multi-output and sparse data are not supported
+   * - `KNeighborsRegressor`
+     - All parameters are supported except:
+
+       - ``algorithm`` != `'brute'`
+       - ``weights`` = `'callable'`
+       - ``metric`` != `'euclidean'` or `'minkowski'` with ``p`` != `2`
+     - Only dense data is supported
+   * - `LinearRegression`
+     - All parameters are supported except:
+
+       - ``normalize`` != `False`
+       - ``sample_weight`` != `None`
+     - Only dense data is supported, `#observations` should be >= `#features`.
+
+Clustering
+**********
+
+.. list-table::
+   :widths: 10 30 20
+   :header-rows: 1
+   :align: left
+
+   * - Algorithm
+     - Parameters & Methods
+     - Data formats
+   * - `KMeans`
+     - All parameters are supported except:
+
+       - ``precompute_distances``
+       - ``sample_weight`` != `None`
+       - ``Init`` = `'k-means++'` fallbacks to CPU.
+     - Sparse data is not supported
+   * - `DBSCAN`
+     - All parameters are supported except:
+
+       - ``metric`` != `'euclidean'`
+       - ``algorithm`` not in [`'brute'`, `'auto'`]
+     - Only dense data is supported
+
+Dimensionality reduction
+************************
+
+.. list-table::
+   :widths: 10 30 20
+   :header-rows: 1
+   :align: left
+
+   * - Algorithm
+     - Parameters & Methods
+     - Data formats
+   * - `PCA`
+     - All parameters are supported except:
+     
+       - ``svd_solver`` not in [`'full'`, `'covariance_eigh'`]
+       - ``fit`` is the only method supported
+     - Sparse data is not supported
+
+Nearest Neighbors
+*****************
+
+.. list-table::
+   :widths: 10 30 20
+   :header-rows: 1
+   :align: left
+
+   * - Algorithm
+     - Parameters
+     - Data formats
+   * - `NearestNeighbors`
+     - All parameters are supported except:
+
+       - ``algorithm`` != `'brute'`
+       - ``weights`` = `'callable'`
+       - ``metric`` not in [`'euclidean'`, `'manhattan'`, `'minkowski'`, `'chebyshev'`, `'cosine'`]
+     - Only dense data is supported
+
+Other tasks
```

Comment:
There are two sections titled "Other Tasks". Is that intentional?

Reply:
Yes, there are the same subsections for each different hardware (CPU, GPU, SPMD)